lib/utils.pm

```diff
@@ -1322,8 +1322,7 @@ sub reconnect_mgmt_console {
     }
     elsif (check_var('ARCH', 'ppc64le')) {
         if (check_var('BACKEND', 'spvm')) {
-            select_console 'novalink-ssh';
-            type_string " mkvterm --id " . get_required_var('NOVALINK_LPAR_ID') . "\n";
+            select_console 'novalink-ssh', await_console => 0;
```

Comment:
why the `await_console => 0`?

Reply:
This was quite tricky to find.   If it is not specified it waits for console http://10.100.12.105/tests/3353#step/lpar/3. It is ok for newly initiated console, but when we are switching back to existing console which has already `mkvterm` running it will fail http://10.100.12.105/tests/3353#step/lpar/11. Also it is used for other consoles in `reconnect_mgmt_console` as well.

Reply:
hm, not sure if this the right way and if we will not hit problems later on then. Well, anyway, ok :)

Reply:
Maybe @Soulofdestiny or @rwx788 could you please have look as well? thank you.